pkgs/games/anki/default.nix

```diff
@@ -167,7 +173,7 @@ buildPythonApplication rec {
       inherit manual;
     };
 
-    meta = with stdenv.lib; {
+    meta = with lib; {
```

Comment:
Unrelated, but what's the advantage of passing `lib` as an input to the derivation?